Moesif.NetFramework.Test/MoesifNetFrameworkTest.cs

```diff
@@ -146,7 +149,7 @@ public void It_Should_Update_User()
                 {"campaign", campaign},
             };
 
-            moesifMiddleware.UpdateUser(user);
+            moesifMiddleware_standalone.UpdateUser(user);

```

Comment:
@qmjava don't we need unit tests for both ctors' methods? Otherwise, the app-based middleware will fail silently like it was failing for standalone-based middelware and we're happy that unit tests were passing.
Merging it for now.

Reply:
you should not be able to call updateUser on a app based middleware because the instance is managed by the app server, and you could not get the instance